const name = 'error';
const invoke = async (error) => {
  console.log('error distube ' + error);
};
export { invoke, name };
